docs/api-gateway-v2/03-04-migration.md

```diff
@@ -180,18 +97,48 @@ rules:
+>**NOTE:** Path definitions in the APIRule CR cannot overlap, otherwise the requests to such paths are rejected. 
 
-  </details>
-</div>
+The **spec.rules** parameter corresponds to **spec.authentication** defined in the Api CR where you can specify a [list multiple JWT token issuers](https://kyma-project.io/docs/components/api-gateway/#details-security-specify-multiple-jwt-token-issuers) to allow secured access and use the **triggerRule.excludedPaths** parameter to exclude paths that don't require any authentication. 
 
+If you want to exclude paths in APIRule definition, for example to avoid two different configurations for a single path, you can use a regular expression with a negative lookahead. In the example, such an expression is used to in the last path exclude the paths already handled by other configuration settings from the `/.*` path. 
 
-The table shows how the APIRule CR's path value corresponds to the **excludedPaths** values for the Api CR, and what negative lookahead value you should add to the `/.*` path:
+Use this table to learn how the APIRule CR's path values correspond to the [**triggerRule.excludedPaths**](/components/api-gateway/#details-security-specify-service-resource-paths-not-secured-with-jwt-authentication) values for the Api CR, and what negative lookahead value you should add to the `/.*` path when defining your APIRule CR.
 
 | Expression type | Description | Sample value | Path value | Negative lookahead value |
 |---|---|---|---|---|
-|**exact**| no changes to be made | `/exact/path/to/resource.jpg` | `/exact/path/to/resource.jpg` | `(?!/exact/path/to/resource.jpg)` |
+|**exact**| no changes | `/exact/path/to/resource.jpg` | `/exact/path/to/resource.jpg` | `(?!/exact/path/to/resource.jpg)` |
 |**prefix**| add `.*` at the end | `/pref/` | `/pref/.*` | `(?!/pref/.*)` |
 |**suffix**| add `.*` at the beginning; in negative lookahead add `\s` at the end | `/suffix.ico` | `.*/suffix.ico | (?!.*/suffix.ico\s)` |
-|**regex**| no changes to be made | `/anything.*` | `/anything.*` | `(?!/anything.*)` |
+|**regex**| no changes | `/anything.*` | `/anything.*` | `(?!/anything.*)` |
+
+If the same **excludedPaths** element is present throughout the authentication settings, a particular path doesn't require any authentication. In that case, create a rule with `handler: allow`, so that you don't have to exclude the path using a negative lookahead.
+ 
+3. When the configuration is ready, create the APIRule object and test if the service is working as expected on the new host. It should work the same on both hosts.
+
+4. Remove the dependent resources of the Api CR:
+
+   ```shell script
+     kubectl delete virtualservice {API_NAME} -n {NAMESPACE}
+    ```
+Make sure that the Virtual Service resource is deleted before proceeding.
+
+5. If the Api CR was secured with an authentication mechanism, delete the Policy resource:
```

Comment:
The Policy CR is also a dependent resource - I would suggest merging it with point 4 or use 
```
4. Remove the VirtualService resource:
```
instead of
```
4. Remove the dependent resources of the Api CR:
```